test/test_payment_initiation.rb

```diff
@@ -4,6 +4,8 @@
 class PlaidPaymentInitiationTest < PlaidTest
   # rubocop:disable Metrics/MethodLength
   def test_all_routes
+    skip 'skipping test that needs vcr updates, passing in beta'
```

Comment:
I'm a little wary of skipping tests, is there something especially difficult here? (I notice you updated VCRs below?). If we need to skip tests, can we file a ticket to update vcr and get these re-enabled?

I realize this client is going to be considered deprecated soon, but it may be a good idea to keep it in a tested state if possible, since many clients are going to be on this branch for a while.

Reply:
ah right meant to mention @nkomanski fixed it in this PR. I will wait for that to merge first and unskip this.

https://github.com/plaid/plaid-ruby/pull/315